app/controllers/miq_policy_controller/policies.rb

```diff
@@ -106,8 +106,8 @@ def policy_copy
                          :target_id    => new_pol.id,
                          :target_class => "MiqPolicy",
                          :userid       => session[:userid],
-                         :message      => _("New Policy ID %{new_id} was copied from Policy ID %{old_id}")) %
-                                            {:new_id => new_pol.id, :old_id => policy.id}
+                         :message      => "New Policy ID %{new_id} was copied from Policy ID %{old_id}" %
+                                          {:new_id => new_pol.id, :old_id => policy.id})
```

Comment:
The right parenthesis end `AuditEvent.success` invocation, so it is correct for it to be there.

Originally, the right parenthesis ended before the percent sign, which was causing the error
described in the bug report.

I also removed the gettext invocation here, since this will become a log message, which
should not be translated to any language.